src/ncar/obs2ioda/src/ncio_mod.f90

```diff
@@ -148,14 +148,17 @@ subroutine write_obs (filedate, write_opt, outdir, itim, fileExt)
       status = netcdfAddVar(netcdfID, trim(ncname), NF90_INT, 1, [trim(ncname)])
 
       do i = 2, n_ncdim
-         status = netcdfAddDim(netcdfID, trim(name_ncdim(i)), val_ncdim(i), ncid_ncdim(i))
-         status = netcdfPutAtt(netcdfID, trim(name_ncdim(i)), val_ncdim(i))
-         status = netcdfAddVar(netcdfID, trim(name_ncdim(i)), NF90_INT, 1, [trim(name_ncdim(i))])
+         if ( .not. trim(name_ncdim(i)) == 'nstring' ) then
+             write(*,*) '  === adding dimension ===> ', name_ncdim(i)
+             status = netcdfAddDim(netcdfID, trim(name_ncdim(i)), val_ncdim(i), ncid_ncdim(i))
+             status = netcdfPutAtt(netcdfID, trim(name_ncdim(i)), val_ncdim(i))
+             status = netcdfAddVar(netcdfID, trim(name_ncdim(i)), NF90_INT, 1, [trim(name_ncdim(i))])
+         end if
       end do
 
       ! define global attributes
-      status = netcdfPutAtt(netcdfID, "min_datetime", xdata(ityp, itim)%min_datetime)
-      status = netcdfPutAtt(netcdfID, "max_datetime", xdata(ityp, itim)%max_datetime)
+!     status = netcdfPutAtt(netcdfID, "min_datetime", xdata(ityp, itim)%min_datetime)
+!     status = netcdfPutAtt(netcdfID, "max_datetime", xdata(ityp, itim)%max_datetime)
```

Comment:
are these used for anything?  would like to remove these entirely

@amstokely or @ibanos90 have you ever seen these Attributes be used these were new to me

@fabiolrdiniz same question?